Frameworks/UIKit/UIViewController.mm

```diff
@@ -2148,6 +2103,17 @@ - (UIModalPresentationStyle)modalPresentationStyle {
 }
 
 /**
+ @Status Caveat
+ @Notes  WinObjC-only extension providing access
+         to UIModalPresentationFormSheet overlay.
+         Not exposed in header (clients must access
+         via performSelector).
+*/
+- (UIView*)_modalOverlayView {
+    return priv->_modalOverlayView;
+}
```

Comment:
That's needed to allow users to interact with the overlay view, typically to dismiss the modal on overlay-touch (by attaching a gesture recognizer). I didn't want to expose this in the public header, instead forcing the user to access via performSelector (since it requires private API on iOS, but it's often done). What about a commit adding dismiss via touch on overlay to the modal example in WOCCatalog? (will also remove the annotations from the method comment while I'm at it)